src/config/envSchema.ts

```diff
@@ -0,0 +1,125 @@
+/**
+ * ╔═══════════════════════════════════════════════════════════════════════════╗
+ * ║                        🌪️  OYA PROTOCOL NODE  🌪️                          ║
+ * ║                    Environment Variable Schema                            ║
+ * ╚═══════════════════════════════════════════════════════════════════════════╝
+ *
+ * Defines the schema for all environment variables required by the Oya node.
+ * This schema is used to validate configuration at startup.
+ *
+ * @packageDocumentation
+ */
+
+import { ethers } from 'ethers'
+import type { EnvVariable } from '../types/setup.js'
+
+/**
+ * Environment variable schema defining all configuration requirements.
+ * Each entry describes a variable's validation rules, transformations, and metadata.
+ */
+export const envSchema: EnvVariable[] = [
+	{
+		name: 'API_BEARER_TOKEN',
+		required: true,
+		type: 'string',
+		description: 'Bearer token for POST endpoint authentication',
+		sensitive: true,
+		validator: (value) =>
+			value.length >= 32 ||
+			'Token should be at least 32 characters for security',
+	},
+	{
+		name: 'DATABASE_URL',
+		required: true,
+		type: 'url',
+		description: 'PostgreSQL connection string',
+		sensitive: true,
+		validator: (value) => {
+			if (
+				!value.startsWith('postgres://') &&
+				!value.startsWith('postgresql://')
+			) {
+				return 'Must be a valid PostgreSQL connection string'
+			}
+			return true
+		},
+	},
+	{
+		name: 'ALCHEMY_API_KEY',
+		required: true,
+		type: 'string',
+		description: 'Alchemy API key for blockchain interactions',
+		sensitive: true,
+		validator: (value) =>
+			value.length === 32 || 'Alchemy API key should be 32 characters',
+	},
+	{
+		name: 'BUNDLE_TRACKER_ADDRESS',
+		required: true,
+		type: 'address',
+		description: 'BundleTracker contract address on Sepolia',
```

Comment:
We should eventually update this to just "BundleTracker contract address" here and in the dist, since the node will ultimately be running against mainnet, as well. But may want to wait until we refactor the node software to target more than one chain. At the beginning it will just be Sepolia, so fine for now.